main.cognitiveservices.tf

```diff
@@ -0,0 +1,25 @@
+resource "azapi_resource" "aiservice" {
+  count = var.kind == "hub" ? 1 : 0
+
+  type = "Microsoft.CognitiveServices/accounts@2024-04-01-preview"
+  body = jsonencode({
+    properties = {
+      publicNetworkAccess = var.is_private ? "Disabled" : "Enabled"
+      apiProperties = {
+        statisticsEnabled = false
+      }
+    }
+    sku = {
+      "name" : "S0",
```

Comment:
Would be good to make this as a parameter in a way we could increase the sky for larger deployments.